.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+Linked PRs: (links to corresponding PRs, optional)
+
+## Changes
+- (changes here)
+
+## TODOs:
+- [ ] +1
+- [ ] Updated README
+- [ ] Updated CHANGELOG
+- [ ] (Other applicable TODOs)
+- [ ] (Feedback from reviewers)
```

Comment:
I think addressing feedback from reviewers is understood in our Mobify culture. It seems redundant to have a checkbox item for it. :)